Changes for next Shutter Network release

ulope opened the pull request:
- The container image name has changed to `ghcr.io/shutter-network/keyper`
- The assets image tag format was changed to be consistent between testnet and mainnet
- A new assets variable `_ASSETS_NETWORK` was introduced and now gets exported in the metrics push
  Also the script function to replace the env var placeholders was removed since vmagent does this automatically (see: https://docs.victoriametrics.com/vmagent/#how-to-collect-metrics-in-prometheus-format)

ulope committed:
Changes for next Shutter Network release

- The container image name has changed to `ghcr.io/shutter-network/keyper`
- The assets image tag format was changed to be consistent between testnet and mainnet
- A new assets variable `_ASSETS_NETWORK` was introduced and now gets exported in the metrics push
  Also the script function to replace the env var placeholders was removed since vmagent does this automatically
  (see: https://docs.victoriametrics.com/vmagent/#how-to-collect-metrics-in-prometheus-format)

ulope committed:
Fix typo in assets image tag

Marketen commented:
Looks fine. @ulope I have built & installed the package with this PR in my dappnode and didnt encounter any problems, but I probably lack context on shutter network in order to test it correctly.

Has anyone else tested this PR in a working dappnode?

ulope commented:
On our side @konradkonrad tried setting it up in an actual dappnode (you talked in chat with him)

konradkonrad committed:
Minimal fix to config parser

The go-shutter-settings config parser choked on two string array fields,
which previously were marked as only string.

konradkonrad committed:
Make SHUTTER_P2P_ADVERTISEADDRESSES an array

konradkonrad committed:
We need to run configure_keyper before init_chain

Otherwise, we will not have the proper db connection for initialization.

konradkonrad committed:
Use environment variables directly

Using the staker script provided functions did not lead to correct
use of form-config provided values for clients.

konradkonrad committed:
Enabled [Metrics] if PUSH_METRICS enabled

Marketen commented:
Part of this PR has been already merged by https://github.com/dappnode/DAppNodePackage-shutter/pull/4
